.github/workflows/ci.yml

```diff
@@ -1,22 +1,24 @@
-name: Node.js CI
-on: [push, pull_request]
+name: CI
+on:
+  - push
+  - pull_request
 jobs:
-  build:
+  test:
+    name: Node.js ${{ matrix.node-version }} on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
-        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+        os:
+          - 'ubuntu-latest'
+          - 'macos-latest'
+          - 'windows-latest'
+        node-version:
+          - '14'
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v1
       with:
-        node-version: '14.x'
-    - uses: actions/cache@v2
```

Comment:
Delete the cache action since it adds unreliability (I don't quite understand how `restore-keys` works and it seems like it falls back to using outdated caches if there's a cache miss) and I don't like all the boilerplate it requires.